includes/admin/class-omise-admin-page.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * @since 4.0
+ */
+class Omise_Admin_Page extends Omise_Setting {
+	/**
+	 * @var array  a set of system messages.
```

Comment:
PHP also has a set ("a sequence of unique values") class, right?

Reply:
I missed this one. Will update to 
```php
/**
 * @var array  of system messages.
 ...
```